README.md

```diff
@@ -48,12 +42,59 @@ TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports
 
 For more details and latest updates, see [TiDB docs](https://docs.pingcap.com/tidb/stable) and [release notes](https://docs.pingcap.com/tidb/dev/release-notes).
 
+## Community
+
+You can join these groups and chats to discuss and ask TiDB related questions:
+
+- [Contributors Mailing list](https://lists.tidb.io/g/contributors)
+- [Slack Channel](https://slack.tidb.io/invite?team=tidb-community&channel=everyone&ref=pingcap-tidb)
+- [Chinese Forum](https://asktug.com)
+
+In addition to that, you may enjoy following:
+
+- [@PingCAP](https://twitter.com/PingCAP) on Twitter
+- Question tagged [#tidb on StackOverflow](https://stackoverflow.com/questions/tagged/tidb)
+- The PingCAP Team [English Blog](https://en.pingcap.com/blog) and [Chinese Blog](https://pingcap.com/blog-cn/)
+- [TiDB Monthly](https://pingcap.com/weekly/)
+
+For support, please contact [PingCAP](http://bit.ly/contact_us_via_github).
```

Comment:
```suggestion
For enterprise support, please contact [PingCAP](http://bit.ly/contact_us_via_github).
```
The link is for enterprise support.

Reply:
And why use short link?

Reply:
I don't think it must be "enterprise" support. Generally speaking I'd like to drop this message because TiDB is a community effort, but let's remain as is for now.

Reply:
> And why use short link?

I have no idea and suggest remain as is.

Reply:
> I don't think it must be "enterprise" support.
Me neither. What I mean is that, when someone wants general non-enterprise support, this link gives the wrong answer :/